scripts/deployToDB.py

```diff
@@ -235,6 +319,17 @@ def get_stale_data(selector, report):
 
 if __name__ == "__main__":
     for selector in SELECTORS:
+        # skipping accounts update for prod environment
```

Comment:
why do we want to skip accounts ci/cd for prod ?

Reply:
We are keeping this to not roll out any account changes to production. Will remove after final release.